example/src/main/java/io/honeycomb/opentelemetry/android/example/MainActivity.kt

```diff
@@ -63,16 +70,18 @@ class MainActivity : ComponentActivity() {
         setContent {
             val currentTab = remember { mutableStateOf(PlaygroundTab.CORE) }
 
-            HoneycombOpenTelemetryAndroidTheme {
-                Scaffold(
-                    modifier = Modifier.fillMaxSize(),
-                    bottomBar = { NavBar(currentTab) },
-                ) { innerPadding ->
-                    Playground(
-                        otelRum,
-                        currentTab,
-                        modifier = Modifier.padding(innerPadding),
-                    )
+            CompositionLocalProvider(LocalOtelComposition provides otelRum) {
```

Comment:
It feels odd to me that we need to do this kinda adhoc for this specific view. 

I almost think the distro should expose this provider and then anything in the distro could use it? Maybe that would something the eventual compose-specific library exposes?